pages/docs/reference/basic-syntax.md

```diff
@@ -602,3 +602,45 @@ fun main(args: Array<String>) {
 </div>
 
 See [Higher-order functions and Lambdas](lambdas.html).
+
+## Creating basic classes and its instances:
+
+<div class="sample" markdown="1">
+
+``` kotlin
+//sampleStart
+fun main(args: Array<String>) {
+    val rectangle: Rectangle = Rectangle (5.0, 2.0) //no 'new' keyword required in Kotlin while initializing
+    val triangle: Triangle = Triangle (3.0, 4.0, 5.0)
+    print( "Area of rectangle is - ${rectangle.getArea()} and it's perimeter is - ${rectangle.getPerimeter()}" )
+    print( "Area of triangle is - ${triangle.getArea()} and it's perimeter is - ${triangle.getPerimeter()}" )
+}
+//sampleEnd
+
+class Rectangle (var height: Double, var length: Double): Shape(4), rectangleProperties {
+    override fun isSquare(): Boolean = length == height
+    override fun getArea(): Double = height*length
+    override fun getPerimeter(): Double = 2*(height + length)
+}
+
+class Triangle (var sideA: Double, var sideB: Double, var sideC: Double): Shape(3) {
+
+    override fun getArea(): Double {
+        var s = (sideA + sideB + sideC)/2
+        return sqrt(s*(s - sideA)*(s - sideB)*(s - sideC))
```

Comment:
This call cannot be resolved when the code is run at try.kotlinlang.org. Perhaps you meant `Math.sqrt(...)`

Reply:
Yes, thanks.